cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -169,13 +169,13 @@ private Set<String> getLimsIusIdsForShesmu(Case kase) {
         .collect(Collectors.toSet());
   }
 
-  private ShesmuCase convertCaseToShesmuCase(Case kase, Map<Long, Assay> assaysById) {
+  private ShesmuCase convertCaseToShesmuCase(Case kase) {
     Optional<LocalDate> completedDate = kase.getRequisition().getFinalReports().stream()
         .map(qc -> qc.getQcDate())
         .max(LocalDate::compareTo);
     return new ShesmuCase.Builder()
-        .assayName(assaysById.get(kase.getAssayId()).getName())
-        .assayVersion(assaysById.get(kase.getAssayId()).getVersion())
+        .assayName(caseData.getAssaysById().get(kase.getAssayId()).getName())
+        .assayVersion(caseData.getAssaysById().get(kase.getAssayId()).getVersion())
```

Comment:
Very minor optimization: Could retrieve the assay above instead of repeating `caseData.getAssaysById().get(kase.getAssayId())` twice